docs/pages/includes/kubernetes-access/rbac.mdx

```diff
@@ -51,4 +51,24 @@ $ tctl create -f kube-user.yaml
 $ tctl create -f user.yaml
 ```
 
-Log out of Teleport and log in again.
\ No newline at end of file
+Create a ClusterRoleBinding for the "basic-viewers" group:
+
+```code
```

Comment:
What would you think of putting the YAML document in a `yaml` snippet, indicating a filename for the YAML file above the `yaml` snippet, then using `kubectl apply -f` with that filename in a separate snippet? While a user would have to run multiple commands instead of one, it would mean that we could take advantage of YAML syntax highlighting in the page. This would also make it easier to see the comments.

Reply:
I love heredocs but I think you're right, the highlighting is better. Easier for users to tinker with the resource file too if they want.